changelog/fragments/crds-with-core-types.yaml

```diff
@@ -0,0 +1,5 @@
+entries:
+    - description: >
+        Fix bug that occurs to fill the example in the process to generate the CSV when the Operator API has Kinds named as core types by starting to check its groups. 
```

Comment:
```suggestion
        Use group and kind instead of only kind when parsing manifests in the CSV generator
        to avoid marshaling them into the wrong Go types.
```

Reply:
Changed as suggested.